code/game/gamemodes/machinegod/machinegod_items.dm

```diff
@@ -0,0 +1,372 @@
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("stabbed", "poked", "jabbed", "torn", "gored")
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	var/summontime = 85 //when that ends, no spear
+
+/obj/item/weapon/spear/clockspear/New()
+	..()
+	processing_objects += src
+
+/obj/item/weapon/spear/clockspear/process()
+	if(summontime <= 0)
+		var/mob/living/M = loc
+		if(istype(M))
+			M.visible_message("<span class='warning'>The spear vanishes from [M]'s hand.</span>", \
+			"<span class='sinister'>The spear vanishes from your hand.</span>")
+		qdel(src)
+	else
+		summontime--
+
+/obj/item/weapon/spear/clockspear/update_wield(mob/user)
+	icon_state = "spearclock[wielded ? 1 : 0]"
+	item_state = "spearclock[wielded ? 1 : 0]"
+	force = wielded ? 12 : 5
+	if(user)
+		user.update_inv_l_hand()
+		user.update_inv_r_hand()
+	return
+
+/obj/item/weapon/spear/clockspear/attack(target as mob, mob/living/user as mob)
+	//if(isclockcult(user)) NOT IMPLEMENTED YET -velardamakar
+	..()
+	var/mob/living/M = target
+	if(!istype(M)) return
+	if(iscultist(target))
+		M.take_organ_damage(wielded ? 38 : 25)
+	if(issilicon(target))
+		M.take_organ_damage(wielded ? 28 : 15)
+	/*else
+		user.Paralyse(5)
+		user << "<span class='warning'>An unexplicable force powerfully repels the spear from [target]!</span>"
+		var/organ = ((user.hand ? "l_":"r_") + "arm")
+		var/datum/organ/external/affecting = user:get_organ(organ)
+		if(affecting.take_damage(rand(force/2, force))) //random amount of damage between half of the spear's force and the full force of the spear.
+			if(iscultist(user))
+				user << "<span class='sinister'>\"You're liable to put your eye out like that.\"</span>"
+				var/organ = ((user.arm ? "l_":"r_") + "arm")
+				var/datum/organ/external/affecting = user:get_organ(organ)
+				user:pain(affecting, 100, force, 1)
+			user.UpdateDamageIcon()*/
+
+/obj/item/weapon/spear/clockspear/pickup(mob/living/user as mob)
+	/*if(!isclockcult(user)) NOT IMPLEMENTED YET -velardamakar
+		user << "<span class='danger'>An overwhelming feeling of dread comes over you as you pick up the cultist's spear. It would be wise to be rid of this weapon quickly.</span>"
+		user.Dizzy(120)*/
+	if(iscultist(user))
+		user << "<span class='sinister'>\"Does a savage like you even know how to use that thing?\"</span>"
+		var/organ = ((user.hand ? "l_":"r_") + "hand")
+		var/datum/organ/external/affecting = user:get_organ(organ)
+		user:pain(affecting, 10, 1, 1)
+
+/obj/item/weapon/spear/clockspear/throw_impact(atom/A, mob/user)
+	..()
+	var/turf/T = get_turf(A)
+	T.turf_animation('icons/obj/clockwork/structures.dmi',"energyoverlay[pick(1,2)]",0,0,MOB_LAYER+1,'sound/weapons/bladeslice.ogg')
+	var/mob/living/M = A
+	if(!istype(M)) return
+	if(iscultist(M))
+		M.take_organ_damage(15)
+		M.Stun(1)
+	if(issilicon(M))
+		M.take_organ_damage(8)
+		M.Stun(2)
+	qdel(src)
+
+/spell/clockspear
```

Comment:
Just use the basic framework for conjuring.
It even does temporary items for you.